docs/ReleaseNotes.md

```diff
@@ -1,6 +1,77 @@
+ - Performance of the the [PX1073](diagnostics/PX1073.md) and [PX1074](diagnostics/PX1074.md) diagnostics has been improved. Multiple runtime error during the analysis of a graph extension were fixed in these diagnostics by removing the redundant analysis of events from the base graph.
+ 
+ #### Code Snippets
+ Acuminator now provides code snippets for DACs, DAC fields, and graph events. A developer can use these code snippets to speed up writing new code and avoid common mistakes.
+ You can start using code snippets by typing "dac" or "_" and selecting the appropriate snippet in the IntelliSense code completion list as shown in the following screenshot.
+ <br/>
+   ![image](https://user-images.githubusercontent.com/7687400/170328654-d343c2da-9659-4e6c-b1c8-c894dad99292.png)<br/>
+ When you select a shortcut from IntelliSense suggestions and press the Tab key, the system inserts the code snippet as shown in the following screenshot.
+ <br/> 
+   ![image](https://user-images.githubusercontent.com/7687400/170331206-ee2f35b8-028f-4d9c-be44-f489783ac4c5.png)<br/>
+   For more details on installing and using the code snippets, see [the CodeSnippets repository on GitHub](https://github.com/Acumatica/CodeSnippets). 
+   
+ #### Improvements in the PX1008 diagnostic 
+ The [PX1008](diagnostics/PX1008.md) diagnostic has been completely reworked. Now it is a data flow diagnostic that goes recursively into methods and checks how the data is transferred to them.
+ The following enhancements have been implemented in the diagnostic:
+   - Checks that adapters of the `PXAdapter` type, passed to a graph or a graph extension action delegates, are not captured in delegate closures by long run and processing delegates
+   - Improvements in diagnostic messages
+   - Checks that custom views and attributes capture graph and adapter references in a closure of a long run delegate
+   - Bug fixes: The diagnostic displayed a warning when capturing a reference to `this` in a delegate even in a class that was not a graph or a graph extension
+ 
+ #### Improvements in the PX1072 diagnostic 
+ The following fixes have been implemented in the [PX1072](diagnostics/PX1072.md) diagnostic:
+   - In static graph members, the declaring graph now is not considered available for usage in Fluent BQL queries
+   - The diagnostic now does not display a warning if a single local graph is available and used in an  Fluent BQL query.
+   - The diagnostic now displayes a warning when a class contains multiple BQL or Fluent BQL queries and there are no available graphs, that is, graphs instances which already exist and can be used by a BQL query.
+ 
+### Fixed Bugs
+In this version of Acuminator, the following bugs have been fixed:
+ - Code fixes provided by Acuminator diagnostics stopped appearing in the UI in the latest release of Visual Studio 2022.
```

Comment:
This is wrong, only two particular code fixes related to the diagnostic suppression stopped working. It was code fix that generated local comment for the diagnostic suppression and code fix that suppressed alert in the Acuminator suppression file:
```suggestion
 - Two Acuminator code fixes, which suppressed Acuminator diagnostics alerts either with a local comment or in the Acuminator suppression file, stopped appearing in the UI in the latest release of Visual Studio 2022.
```